doc/operation-and-maintenance/MongooseIM-metrics.md

```diff
@@ -6,7 +6,7 @@ MongooseIM uses [ESL's fork of this project](https://github.com/esl/exometer/tre
 
 All metrics are divided into the following groups:
 
-* Per host metrics: Gathered separately for every XMPP host supported by the cluster.
+* Per host type metrics: Gathered separately for every host type supported by the cluster.
  **Warning:** If a cluster supports many (thousands or more) domains, performance issues might occur.
  To avoid this, use global equivalents of the metrics with `all_metrics_are_global` config option.
```

Comment:
That warning is not exactly true anymore, it might as well be a "if a cluster supports many many host types", which, who the heck would do that? 🤷🏽‍♂️

Reply:
Yeah, that is true, I was also wondering about this line. My reasoning was, well, if there is no possibility of performance issues occurring... why do we keep the `all_metrics_are_global` actually? What would you propose writing here? Something like just "There is also a way to group metrics for all host types using the `all_metrics_are_global` config option" seems weak.

Reply:
I was thinking the same, but it is still possible to configure thousands of host_types, even if nobody should ever do that. Dunno, maybe somebody do want to serve 5k domains and have metrics granular for each, so he configures 5k host_types with just one domain each... and if somebody actually wants that then the `all_metrics_are_global` does nothing for him, so yeah, I'm failing to see why do we still want to keep that flag 🤔 Let's ask the rest of the team 🤔